packages/itwinui-css/src/stepper/stepper.scss

```diff
@@ -4,114 +4,105 @@
-    overflow-wrap: break-word;
-    user-select: none;
-    transition: background-color $iui-speed-fast ease-out, border-color $iui-speed-fast ease-out,
-      color $iui-speed-fast ease-out;
-    border: 1px solid var(--iui-color-foreground-positive);
-    background-color: var(--iui-color-background-1);
-    color: var(--iui-color-foreground-positive);
+  &:first-of-type {
+    --_iui-stepper-step-track-before-color: transparent;
   }
 
+  &:last-of-type {
+    --_iui-stepper-step-track-after-color: transparent;
+  }
+}
+
+@mixin iui-stepper-step-modifiers {
   // Current step
   &.iui-current {
-    font-weight: $iui-font-weight-semibold;
+    --_iui-stepper-step-background-color: var(--iui-color-background-positive);
+    --_iui-stepper-step-border-color: var(--iui-color-background-positive);
+    --_iui-stepper-step-number-color: var(--iui-color-foreground-accessory);
+    --_iui-stepper-step-track-after-color: var(--iui-color-background-border);
 
-    .iui-stepper-step-name {
-      color: var(--iui-color-foreground-positive);
-    }
-
-    .iui-stepper-circle {
-      color: var(--iui-color-foreground-accessory);
-      border-color: var(--iui-color-background-positive);
-      background-color: var(--iui-color-background-positive);
-    }
-
-    .iui-stepper-track-content::after {
-      background-color: var(--iui-color-background-5);
-    }
+    font-weight: $iui-font-weight-semibold;
   }
 
   // Future steps
   &.iui-current ~ & {
-    .iui-stepper-circle {
-      border: 1px solid var(--iui-color-background-border);
-      background-color: var(--iui-color-background-1);
-      color: var(--iui-text-color-muted);
-    }
-
-    .iui-stepper-step-name {
-      color: var(--iui-text-color-muted);
-    }
-
-    .iui-stepper-track-content::before,
-    &:not(:last-of-type) .iui-stepper-track-content::after {
-      background-color: var(--iui-color-background-5);
+    --_iui-stepper-step-background-color: var(--iui-color-background-1);
+    --_iui-stepper-step-border-color: var(--iui-color-background-border);
+    --_iui-stepper-step-number-color: var(--iui-text-color-muted);
+    --_iui-stepper-step-text-color: var(--iui-text-color-muted);
+    --_iui-stepper-step-track-before-color: var(--iui-color-background-5);
+
+    &:not(:last-of-type) {
+      --_iui-stepper-step-track-after-color: var(--iui-color-background-5);
     }
   }
 
   // Clickable steps
-  &.iui-clickable {
-    .iui-stepper-circle {
-      cursor: pointer;
-      @media (prefers-reduced-motion: no-preference) {
-        transition: background-color $iui-speed-fast ease-out;
-      }
-
-      &:hover {
-        color: var(--iui-color-foreground-accessory);
-        border: 1px solid var(--iui-color-background-positive);
-        background-color: var(--iui-color-background-positive);
-      }
-    }
+  &.iui-clickable:focus {
+    outline: 0;
+  }
+}
 
-    &:focus {
-      outline: 0;
+@mixin iui-stepper-circle {
+  align-items: center;
+  border-radius: 50%;
```

Comment:
should use `border-radius-round` here?

Reply:
Yes we can use `$iui-border-radius-round` here, replaced.